### PR TITLE
Fix unsupported asyncio syntax

### LINKED ISSUE #7979: shutdown-idle failed for fedora-37
### Qubes OS release
R4.1

### Brief summary
With the  fedora-37 TemplateVM, the shutdown-idle service doesn't shutdown the AppVM.

### Steps to reproduce
- Install the fedora-37 template from templates-itl-testing.
- Install the `qubes-app-shutdown-idle` package (1.0.8-1.fc37) in the template
- create a fedora-37 based AppVM : test-f37
- activate shutdown-idle service from dom0 : qvm-features test-f37 service.shutdown-idle 1
- start test-f37, run a terminal, close the terminal
- wait 15mn

### Expected behavior
The AppVM shutdowns after 15mn without activities.

### Actual behavior
The AppVM doesn't shutdown after 15mn without activities.

Related to
- [Fedora 37 template #7807](https://github.com/QubesOS/qubes-issues/issues/7807)

### PR DESCRIPTION
**Pull request by marmarta**
Wrap coroutines into explicit tasks.

fixes QubesOS/qubes-issues#7979

### CODE REVIEW

**Comment by marmarta** [comment]
PipelineRetryFailed

**Comment by marmarta** [comment]
PipelineRetry

**Comment by marmarek** [comment]
See https://github.com/QubesOS/qubes-core-admin/pull/508#issuecomment-1380686332

File: qubesidle/idleness_monitor.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ async def monitor_idleness(self):
                 await self.watch_the_watchers
             tasks = []
             for w in self.watchers:
-                tasks.append(w.wait_for_state_change())
+                tasks.append(asyncio.Task(w.wait_for_state_change()))
```
Comment (DemiMarie) [inline qubesidle/idleness_monitor.py]:
```suggestion
                tasks.append(asyncio.create_task(w.wait_for_state_change()))
```

**Review by DemiMarie — CHANGES_REQUESTED** [review]

File: qubesidle/idleness_monitor.py (hunk 1 above)
Comment (marmarta) [reply qubesidle/idleness_monitor.py]: Thanks!

**Review by DemiMarie — APPROVED** [review]

File: qubesidle/idleness_monitor.py (hunk 1 above)
Comment (DemiMarie) [reply qubesidle/idleness_monitor.py]: You’re welcome!

**Comment by marmarek** [comment]
The same needs to be fixed in tests too (see https://gitlab.com/QubesOS/qubes-app-shutdown-idle/-/jobs/3600375702 for example).

**Comment by marmarta** [comment]
PipelineRetryFailed

**Comment by marmarta** [comment]
centos fails for some weird reason, but it doesn't look like my fault. Tests are fixed, though

**Pull request merged**